thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/utils/RequestStatistics.java

```diff
@@ -0,0 +1,157 @@
+package com.linkedin.thirdeye.anomaly.utils;
+
+import java.util.Map;
+
+
+/**
+ * Statistics container for data source request performance log
+ */
+public final class RequestStatistics {
+  Map<String, Long> requestsPerDatasource;
```

Comment:
Make them private.

Reply:
they're default and accessed from the package, e.g. by the formatter